app/editor/src/components/toggle-group/styled/ToggleGroup.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import { IRowProps, Row } from 'tno-core';
+
+export interface IStyledToggleGroupProps extends IRowProps {
```

Comment:
Don't put an interface in this file.  Keep it consistent with other styled files.

Reply:
Don't even need that prop anymore :)